build.gradle

```diff
@@ -125,6 +125,7 @@ dependencies {
   compile group: 'org.twitter4j', name: 'twitter4j-core', version: '4.0.2'
   compile group: 'org.twitter4j', name: 'twitter4j-stream', version: '4.0.2'
   compile group: 'xerces', name: 'xerces', version: '2.4.0'
+  compile fileTree(dir: 'libs', include: ['*.jar'])
```

Comment:
The intention over time was to remove the `libs/` folder. I notice that you've added boilerpipe to this, There are some problems with that library. It's not being maintained anymore and a better thing to do right now is to take the maintainer status of that project onto the loklak org and publish a gradle entry and compile via the gradle name.